usrtools/txgen/app/cli-functions.c

```diff
@@ -276,14 +347,21 @@ pcap_cmd(int argc, char **argv)
     return 0;
 }
 
-static struct cli_map start_map[] = {{10, "start %P"}, {20, "stop %P"}, {-1, NULL}};
+static struct cli_map start_map[] = {{10, "start %P"},
+                                     {20, "stop %P"},
+                                     {50, "start %P latsampler"},
+                                     {60, "stop %P latsampler"},
```

Comment:
Normally in txgen you enable/disable features on a port then start/stop the port. This way the port can be configured with enable/disable/set commands then you can save the configuration for later. You could remove these commands and use enable/disable commands i.e. 'enable/disable <portlist> latency'

If you want to save the latency configuration you will need to output all of the information in the save command. It would be good to save the data, I do not think pktgen did it that way :-(, so if you want it is not a requirement.

Reply:
Looks like this was the way it was done in Pktgen :-(, so we can leave it this way.

Reply:
No worries I can have a look at this once I have a full suit of clean tests :) 

I'm going to close this review for now till I have everything updated, then I will re-open

I've addressed the rest of the comments. Thanks for reviewing